include/dxc/Support/HLSLOptions.h

```diff
@@ -165,7 +165,7 @@ class DxcOpts {
   bool EnableDX9CompatMode = false;     // OPT_Gec
   bool EnableFXCCompatMode = false;     // internal flag
   LangStd HLSLVersion = LangStd::vUnset; // OPT_hlsl_version (2015-2021)
-  bool Enable16BitTypes = false; // OPT_enable_16bit_types
+  bool Enable16BitTypes = false;         // OPT_enable_16bit_types
```

Comment:
While I think the formatting change here is a strict improvement, maybe leave it out for this commit since it is unrelated.